### PR TITLE
adding another option for path check

### PR DESCRIPTION
**Pull request by maimorag**
## Related Issues
fixes: None

## Description
Images in path Packs/ShiftManagement/doc_files couldn't been access, after adding another option by checking the path with ShiftManagement/doc_files  only it managed to find the file- It look like concatenate "/Packs" twice in path_validate.convert

### CODE REVIEW

**Comment by maimorag** [comment]
Tests::
1. `Packs/ShiftManagement/Playbooks/Assign_Active_Incidents_to_Next_Shift_V2.yml` which has existing image:
/Users/mmorag/dev/demisto/content/Packs/ShiftManagement/doc_files/Assign_Active_Incidents_to_Next_Shift_V2.png
got:
`All validations passed.`

2.  `Packs/ShiftManagement/Playbooks/Assign_Active_Incidents_to_Next_Shift_V2.yml` which has not existing image:
/Users/mmorag/dev/demisto/content/Packs/ShiftManagement/doc_files/Assign_Active_Incidents_to_Next_Shift_V2_not_exist.png
got:
`Packs/ShiftManagement/Playbooks/Assign_Active_Incidents_to_Next_Shift_V2.yml: [RM114] - The following images do not exist or have additional characters present in their declaration within the README: Packs/ShiftManagement/doc_files/Assign_Active_Incidents_to_Next_Shift_V2_not exists.png
`

File: demisto_sdk/commands/validate/validators/RM_validators/RM114_is_image_exists_in_readme.py [hunk 1]
```diff
@@ -76,6 +76,10 @@ def get_invalid_image_paths(pack_name: str, image_paths: List[str]) -> List[str]
                 path_validate.convert(image_path, param=None, ctx=None)
 
             except click.exceptions.BadParameter:
-                invalid_image_paths.append(image_path)
+                try:
+                    alternative_path = image_path[len("Packs/") :]
```
Comment (BEAdi) [inline demisto_sdk/commands/validate/validators/RM_validators/RM114_is_image_exists_in_readme.py:80]:
```suggestion
                    alternative_path = image_path.removeprefix("Packs/")
```
More readable

File: demisto_sdk/commands/validate/validators/RM_validators/RM114_is_image_exists_in_readme.py [hunk 2]
```diff
@@ -66,7 +66,7 @@ def get_invalid_image_paths(pack_name: str, image_paths: List[str]) -> List[str]
         Returns:
             List[Path]: A list of invalid image files full paths.
         """
-        path_validate = click.Path(exists=True, dir_okay=False)
+        path_validate = click.Path(exists=True, dir_okay=False, allow_dash=True)
```
Comment (BEAdi) [inline demisto_sdk/commands/validate/validators/RM_validators/RM114_is_image_exists_in_readme.py:69]: This is needed? Or can it work without it too?

File: .changelog/4717.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Fixed an issue where RM114 failed since it concatenate "/Packs" twice.
```
Comment (BEAdi) [inline .changelog/4717.yml:2]:
```suggestion
- description: Fixed an issue where RM114 falsely failed when it concatenated "Packs/" twice to the file path.
```

**Review by BEAdi — APPROVED** [review]
Great job!

**Pull request merged**